webstorage/event_constructor.html

```diff
@@ -9,21 +9,18 @@
     <h1>event_session_Constructor</h1>
     <div id="log"></div>
     <script>
-        test(function() {
-            var t = async_test("storageeventinit test");
+        async_test(function(t) {
```

Comment:
This was a drive-by change to make the test more similar event_constructor_eventinit.html. All the business with `dispatchEvent` actually isn't necessary to test the constructor, but I just wanted to get of the extra no-op test.

Reply:
This can be a synchronous test as you're not actually waiting for anything. You still need to wrap the callbacks as you've done though.

Reply:
I know, but the whole `dispatchEvent` thing is also unnecessary. This made it consistent with other tests, and I don't want to rewrite them all since it's pretty harmless.